festflow/views.py

```diff
@@ -109,3 +112,30 @@ def complete_profile(request):
     context['backend'] = backend
     context['profile_form'] = profile_form
     return render(request, 'festflow/complete_profile.html', context)
+
+
+def subscribe(request):
+    context = {}
+
+    if request.method == 'POST':
+        form = SubscriptionForm(request.POST)
+        to_addr = request.POST.get('contact_email', '')
+        if Subscription.objects.filter(contact_email=to_addr).count():
```

Comment:
Take a look [here](http://stackoverflow.com/questions/6868939/in-django-what-is-the-most-efficient-way-to-check-for-an-empty-query-set) bro @ghoshbishakh